https://redhat.atlassian.net/browse/ACM-31308 separating more files part 3 (#8884)
open-cluster-management/rhacm-docs: observability/obs_forward_alerts.adoc

:_mod-docs-content-type: PROCEDURE

[#obs-forward-alerts]
= Forwarding alerts

[role="_abstract"]

> After you enable observability, alerts from your {ocp-short} managed clusters are automatically sent to Alertmanager on your hub cluster. By default, all platform alerts are sent to Alertmanager on your hub cluster. When you enable user workload alerts on your {ocp-short} managed clusters, user workload alerts are also sent to your hub cluster. 

Reviewer: We don't talk about forwarding (verb) in the description?

I am gathering from this that we need to be clearer in the last line:

_Enable user workload alerts on your {ocp-short} managed clusters to get user workload alerts forwarded to your hub cluster._ 

???

Author: I agree with your suggestion and I would even add updating the file name and doc name

Author: @swopebe let me know your thoughts about updating "Forwarding alerts" to "Enabling user workload alerts" 

and the file name obs_forward_alerts to obs_enable_uw-alerts

Author: During live review: Change to Forwarding your alerts and check link references

> After you enable observability, alerts from your {ocp-short} managed clusters are automatically sent to Alertmanager on your hub cluster. By default, all platform alerts are sent to Alertmanager on your hub cluster. When you enable user workload alerts on your {ocp-short} managed clusters, user workload alerts are also sent to your hub cluster. 

Author: ```suggestion
After you enable observability, alerts from your {ocp-short} managed clusters are automatically sent to Alertmanager on your hub cluster. By default, all platform alerts are sent to Alertmanager on your hub cluster. Enable user workload alerts on your {ocp-short} managed clusters to get user workload alerts forwarded to your hub cluster. 
```
